### ISSUE TITLE
Timers reset on first time the dark mode is activated...

### ISSUE THREAD

**Issue by H3AR7B3A7** [Mrjsaw/ChessClock#21]
To recreate issue:
1. start klok
2. change theme
3. return to main en start again

compare with:
going through steps 1 through 3 again ...

enkel eerste keer resetten de timers

**Comment by Mrjsaw** [comment]
Does changing the theme the first time recreate the SettingsActivity or MainActivity?
recreating SettingsActivity sets the sharedPrefs to the last entered values (if none are given it takes the default values)
recreating MainActivity sets the timers to the sharedPrefs values (TOP_TIME, BOT_TIME)

**Comment by H3AR7B3A7** [comment]
Ja, SettingsActivity, maar dat doet hij iedere keer als ge op die knop duwt ... niet enkel eerste keer. Dan killt hij activity met: 
this.activity?.recreate()
Ik denk niet dat het daar aan ligt.

Maar als ik in MainActivity 'loadData()' uitcomment dan gebeurt het nimeer...

Misschien moeten de prefs van de clock in een andere instantie van SharedPreferences worden bijgehouden en niet worden meegegeven aan onResume? Maar ik dacht dat die enkele dagen geleden nog niet in de sharedPreferences zaten? Ofwel?
